entrypoint.sh

```diff
@@ -6,7 +6,9 @@ export BALENARC_BALENA_URL
 
 set -e
 
-[ -z "${BALENA_TOKEN}" ] && echo "BALENA_TOKEN input cannot be empty" && exit 1
+BALENA_TOKEN=${BALENA_API_KEY:=${BALENA_TOKEN:=}}
```

Comment:
This doesn't change what we support as an input at all, since the env var comes from https://github.com/balena-io/deploy-to-balena-action/blob/v0.17.0/action.yml#L67 and is never going to be anything other than `BALENA_TOKEN`